api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -0,0 +1,95 @@
+package org.pmiops.workbench.db.model;
+
+import java.util.List;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "institution")
+public class DbInstitution {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_id")
+  private long institutionId; // primary opaque key for DB use only
+
+  @Column(name = "api_id", nullable = false, unique = true)
+  private String apiId; // unique key exposed to API
+
+  @Column(name = "long_name", nullable = false)
+  private String longName;
+
+  @Column(name = "organization_type_enum")
+  private Short organizationTypeEnum;
+
+  @Column(name = "organization_type_other_text")
+  private String organizationTypeOtherText;
+
+  @OneToMany(mappedBy = "institution", fetch = FetchType.LAZY)
+  private List<DbInstitutionEmailDomain> emailDomains;
+
+  @OneToMany(mappedBy = "institution", fetch = FetchType.LAZY)
+  private List<DbInstitutionEmailAddress> emailAddresses;
```

Comment:
aside: do we need/want a mapping between email addresses and domains? Like would you want to do a report with all the institutions, then their domains, then the addresses? Not needed yet I guess.

Reply:
Dunno, I'm not seeing it.  Certainly not for existing stories.